src/spacegame3/SpaceGame.java

```diff
@@ -31,6 +31,9 @@
     }
 
 
+    /*
+        *Instance variables
+    */
```

Comment:
You can use /** */ on each field to say what they represents/does if it is important for outsiders to know about those fields (since it will go in the javadoc) which in this case i dont think is necessary or  you could use // either before the field or at the end of its line to say the same for developers so they what what use the field has